project/plugins.sbt

```diff
@@ -5,3 +5,4 @@ addSbtPlugin("ch.epfl.scala" % "sbt-version-policy" % "2.0.1")
 addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.3.2")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.9.3")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.34")
+addSbtPlugin("com.eed3si9n" % "sbt-projectmatrix" % "0.9.0")
```

Comment:
see https://github.com/scalacenter/scalafix.g8/pull/19 for some prior discussions